code/game/machinery/doors/airlock.dm

```diff
@@ -459,6 +498,28 @@ var/list/airlock_overlays = list()
 	if(charge && panel_open)
 		user << "<span class='warning'>Something is wired up to the airlock's electronics!</span>"
 
+	if(panel_open)
+		switch(security_level)
+			if(AIRLOCK_SECURITY_NONE)
+				user << "Wires are exposed!"
+			if(AIRLOCK_SECURITY_METAL)
+				user << "There is welded metal cover protecting wires"
+			if(AIRLOCK_SECURITY_PLASTEEL_I_S)
+				user << "There is cut plasteel"
+			if(AIRLOCK_SECURITY_PLASTEEL_I)
+				user << "There is inner welded plasteel cover"
+			if(AIRLOCK_SECURITY_PLASTEEL_O_S)
+				user << "There is cut plasteel"
+			if(AIRLOCK_SECURITY_PLASTEEL_O)
+				user << "There is welded plasteel cover"
+			if(AIRLOCK_SECURITY_PLASTEEL)
+				user << "There is protective grile over panel"
```

Comment:
grille
There should also be articles here and in other messages in this switch.